cmd/shfmt/main.go

```diff
@@ -168,7 +169,8 @@ Printer options:
 Utilities:
 
   -f, --find   recursively find all shell files and print the paths
-  --tojson     print syntax tree to stdout as a typed JSON
+  --to-json    print syntax tree to stdout as a typed JSON
```

Comment:
does it make sense to mention here the flag change, eg `available since v3`? perhaps to make the change obvious.

Reply:
We generally record that info via the changelog. I'm not sure whether including that in the help text or man page helps. If you run `shfmt -h` and the flag is there, then you can use it. If the flag isn't there, then your version is too old, so you can't use it.